tests/test_form.py

```diff
@@ -129,6 +131,17 @@ def test_get_metadata(self):
         actual = form.get_metadata(metadata_keys, form_object)
         expected = u'<!-- @sky: blue -->\n<!-- @earth: None -->\n'
         self.assertEqual(actual, expected)
+        form_object = MultiDict([
```

Comment:
(using a MultiDict here is just me being lazy and copying some of the values the app sends...rather than re-formatting into a plain old dict)

Reply:
just fyi if you need again in the future, (I don't think it creates an issue here apart of the dependency.) `MultiDict()` has `form_object.to_dict()`, which converts to a normal dictionary.

```python
>>> form_object.to_dict()
{'url': u'http://localhost:5000/issues/new', 'reported_with': u'desktop-reporter', 'ua_header': u'Mozilla/5.0...Firefox 59.0', 'extra_labels': [u'type-stylo', u'type-webrender-enabled'], 'browser': u'Firefox 59.0'}
```

Reply:
Interesting to see we had a missing test.